app/src/server/services/email/charge.ts

```diff
@@ -840,16 +840,20 @@ export const sendChargeTemplate = async () => {
     .catch(error => console.error(error));
 }
 
-export const sendEmail = async (receiverEmail: string, fields: {chargerUsername: string, homeName: string, amountOwed: string, dueDate: Date, comment: string}) => {
+export const sendEmail = async (receiverEmail: string, fields: {chargerUsername: string, homeName: string, amountOwed: string, dueDate: Date, address: string}) => {
```

Comment:
I think we want Issue Date and Comment. I think users would want to know traceability-wise what they are paying for. It should also contain the charger Email where maybe they want to email the user back for some reason.